src/seedu/addressbook/ui/TextUi.java

```diff
@@ -91,28 +79,28 @@ public String getUserCommand() {
     public void showWelcomeMessage(String version, String storageFilePath) {
         String storageFileInfo = String.format(MESSAGE_USING_STORAGE_FILE, storageFilePath);
         showToUser(
-                DIVIDER,
-                DIVIDER,
+                Formatter.DIVIDER,
+                Formatter.DIVIDER,
                 MESSAGE_WELCOME,
                 version,
                 MESSAGE_PROGRAM_LAUNCH_ARGS_USAGE,
                 storageFileInfo,
-                DIVIDER);
+                Formatter.DIVIDER);
```

Comment:
Do you think that you can further abstract this method out?
This code seems to be formatting the welcome message by displaying `MESSAGE_WELCOME`, `version`, so and so forth.